README.md

```diff
@@ -50,7 +62,7 @@ Nothing happens yet. The script exports a global object `off` to communicate wit
 </script>
 ```
 
-Notice calling [`off.install()`](https://rawgit.com/lodr/offliner/concept/docs/classes/OfflinerClient.html#method_install) will install a worker assuming it's called `offliner-worker.js` which is expected to be at the root of your server. If this is not the case use `data-root` and `data-worker` to change the base directory where your worker is and the name of your worker. Remember the worker will be registerd with `data-root` as scope so it will control all the resources under this path.
+Calling [`off.install()`](https://rawgit.com/lodr/offliner/concept/docs/classes/OfflinerClient.html#method_install) expects a worker called `offliner-worker.js` at the root of your server. If this is not the case use `data-root` and `data-worker` to change the base directory where your worker is and the name of your worker. Remember the worker will be registerd with `data-root` as scope so it will control all the resources under this path.
```

Comment:
To avoid the strange formatting of the documentation pages, we should change the protocol of the docs to `http` instead of `https`.